packages/docs/content/guide/kr/03. block-types/01. block-type-guide-intro.md

```diff
@@ -0,0 +1,242 @@
+---
+group: "block-types"
+order: 1
+title: "블록 타입 가이드 소개"
+slug: "block-type-guide-intro"
+description: "React-Notion-Custom에서 지원하는 블록 타입들의 개요, 사용 방법, 커스터마이징 옵션, 그리고 내부 동작 원리를 소개합니다."
+---
+
+# 블록 타입 가이드 소개
+
+React-Notion-Custom은 Notion의 다양한 블록 타입을 지원하며, 각 블록 타입에 대해 Notion과 최대한 유사한 스타일의 컴포넌트를 제공합니다. 이 가이드에서는 현재 지원되는 블록 타입들, 각 블록 타입의 Props, 커스터마이징 방법, 그리고 내부 동작 원리에 대해 설명합니다.
+
+## 지원되는 블록 타입
```

Comment:
p0: The list of 'supported block types' mentioned in the official documentation tends to change frequently, and it seems likely that the current markdown format will require manual updates.
To address this, it might be worth considering automation or exploring more efficient methods.

Reply:
Honestly I have no idea about automation. Do you have any idea? I think someday updating will be less

Reply:
I agree with you that there will be fewer updates later on.
I thought it could be automated, so I wrote it down. Actually, I don't have a specific plan, so I'll share it if I find out.

Reply:
Thank you

Reply:
I made a mistake with the code review priority numbers....! I fixed it.